app/uk/gov/hmrc/serviceconfigs/persistence/DeploymentConfigRepository.scala

```diff
@@ -16,59 +16,38 @@
 
 package uk.gov.hmrc.serviceconfigs.persistence
 
-import com.mongodb.BasicDBObject
 import javax.inject.{Inject, Singleton}
+import com.mongodb.BasicDBObject
 import org.mongodb.scala.bson.BsonDocument
-import org.mongodb.scala.model.Filters.{and, equal, in}
-import org.mongodb.scala.model.Projections.include
+import org.mongodb.scala.model.Filters.{and, equal}
 import org.mongodb.scala.model._
 import uk.gov.hmrc.mongo.MongoComponent
 import uk.gov.hmrc.mongo.play.json.PlayMongoRepository
+import uk.gov.hmrc.mongo.transaction.{TransactionConfiguration, Transactions}
 import uk.gov.hmrc.serviceconfigs.model.{DeploymentConfig, Environment}
 
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class DeploymentConfigRepository @Inject()(
-  mongoComponent: MongoComponent
+  final val mongoComponent: MongoComponent
```

Comment:
`override` to make it clear why it's a val?

Reply:
Thanks changed